src/rest/routes/github-callback/index.ts

```diff
@@ -49,8 +49,10 @@ const getPostMessageScript = function(opts: Record<string, unknown>) {
 	return `<html>
 			<body></body>
 			<script>
+			if (window.opener) {
```

Comment:
Doesn't it always have that `window.opener`?
And in your change, what if it doesn't have `window.opener`, what UI error will bring user to?

Reply:
When we are requesting to install an org from Github, then the admin receives an email with the link from Github. When dealing with that link, then at that time this `window.opener` is undefined, which causes the response screen to be a JS error screen, but the installation is done. That's why I added this check.
We do not have any UI for showing the error at the moment, we can do that in another task. The objective here is to not show any JS error.